scripts/helpers/time_series_helpers.py

```diff
@@ -2,8 +2,10 @@
 Functions to support development of time series analytics work.
 """
 import datetime
-from typing import Tuple, Union
+import io
+from typing import Tuple
```

Comment:
I think you don't need this anymore if you use tuple throughout rather than Tuple - sorry, realise this one came from me in the first place!